packages/algoliasearch/src/builds/node.ts

```diff
@@ -609,6 +635,41 @@ export type SearchClient = BaseSearchClient & {
     restrictions: SecuredApiKeyRestrictions
   ) => string;
   readonly getSecuredApiKeyRemainingValidity: (securedApiKey: string) => number;
+  readonly clearDictionaryEntries: (
+    dictionary: string,
+    requestOptions?: RequestOptions & SaveDictionaryEntriesOptions
+  ) => Readonly<WaitablePromise<SaveDictionaryEntriesResponse>>;
+  readonly deleteDictionaryEntries: (
+    dictionary: string,
+    objectIDs: readonly string[],
+    requestOptions?: RequestOptions & SaveDictionaryEntriesOptions
+  ) => Readonly<WaitablePromise<SaveDictionaryEntriesResponse>>;
+  readonly replaceDictionaryEntries: (
+    dictionary: string,
+    entries: readonly DictionaryEntry[],
+    requestOptions?: RequestOptions & SaveDictionaryEntriesOptions
+  ) => Readonly<WaitablePromise<SaveDictionaryEntriesResponse>>;
+  readonly saveDictionaryEntries: (
+    dictionary: string,
+    entries: readonly DictionaryEntry[],
+    requestOptions?: RequestOptions & SaveDictionaryEntriesOptions
+  ) => Readonly<WaitablePromise<SaveDictionaryEntriesResponse>>;
+  readonly searchDictionaryEntries: (
+    dictionary: string,
+    query: string,
+    requestOptions?: RequestOptions
+  ) => Readonly<Promise<SearchDictionaryEntriesResponse>>;
+  readonly getDictionarySettings: (
+    requestOptions?: RequestOptions
+  ) => Readonly<Promise<GetDictionarySettingsResponse>>;
+  readonly setDictionarySettings: (
+    settings: readonly DictionarySettings[],
+    requestOptions?: RequestOptions
+  ) => Readonly<WaitablePromise<SetDictionarySettingsResponse>>;
+  readonly getDictionaryTask: (
+    taskID: number,
+    requestOptions?: RequestOptions
+  ) => Readonly<Promise<TaskStatusResponse>>;
```

Comment:
what's this method for? should it not be a generic waitTask?

Reply:
it should indeed but I had issues with the fact there were two methods named the same way so I renamed it. It should however be changed for the generic waitTask indeed